weld/subsystem/src/main/java/org/jboss/as/weld/deployment/processors/WeldDeploymentProcessor.java

```diff
@@ -324,12 +325,16 @@ public void undeploy(DeploymentUnit context) {
         }
     }
 
-    static List<SetupAction> getSetupActions(DeploymentUnit deploymentUnit) {
+    public static List<SetupAction> getSetupActions(DeploymentUnit deploymentUnit) {
```

Comment:
I haven't digested the important part of this yet, but AFAICT this visibility change is not needed.

Reply:
correct, at some point had the new processor in different package and missed this was not needed after all cleanup, fixed now